src/View/Includes/_btnToGoUp.html.twig

```diff
@@ -0,0 +1,3 @@
+<a href="" class="btn-up"><svg xmlns="http://www.w3.org/2000/svg" width="52" height="50" fill="var(--light)" class="bi bi-arrow-up rounded p-2 position-fixed bottom-0 end-0 me-3 mb-3 d-none d-sm-none d-md-block" viewBox="0 0 16 16" style="background-color:var(--primary-color)">
+  <path fill-rule="evenodd" d="M8 15a.5.5 0 0 0 .5-.5V2.707l3.146 3.147a.5.5 0 0 0 .708-.708l-4-4a.5.5 0 0 0-.708 0l-4 4a.5.5 0 1 0 .708.708L7.5 2.707V14.5a.5.5 0 0 0 .5.5z"/>
+</svg></a>
```

Comment:
"_btnToGoUp" => not a valid file name (abbreviation + should be snake_case)

Reply:
ok correction done